content/en/logs/log_collection/csharp.md

```diff
@@ -293,20 +299,141 @@ logs:
 
 That's it! Now, all your logs are going to be in proper JSON automatically understood by your Datadog application.
 
-## Agentless logging
+## Agentless logging with APM
 
-It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
-{{< tabs >}}
-{{% tab "Serilog" %}}
+It is possible to stream logs from your application to Datadog directly, without making any code changes, using the .NET APM automatic instrumentation library. This approach sends logs directly to Datadog, so it does not benefit from [features such as sensitive data scrubbing][3] which are provided by the Datadog Agent. For that reason, we recommend using file tail logging where possible, but it is useful in environments where this is not possible (when using [Azure App Service][4] for example).
```

Comment:
It should be pointed out that users will still be able to rely on our server side scrubbing capabilities performed by Sensitive Data Scanner (with link to its page: https://docs.datadoghq.com/account_management/org_settings/sensitive_data_detection/#overview)

Reply:
Added in https://github.com/DataDog/documentation/pull/13907/commits/1a70bc07eeaf48a2d0e0d514ac2e270405e25364.